.github/workflows/package-tests.yml

```diff
@@ -11,7 +11,6 @@ on:
       - main
       - master
       - dev
-  workflow_call:
```

Comment:
This line needs to come back. It allows this workflow to be called from other workflows, e.g. from the mm-tools and tabular-tools repositories.